### PR TITLE
docs: update schema of request

### LINKED ISSUE #150: Update schema of Request
The schema for JSRequest has changed https://github.com/tailcallhq/tailcall/pull/1409

Update the documentation with these changes.

### PR DESCRIPTION
**Pull request by guptadeepak8**
fix: #150
/claim #150

### CODE REVIEW

File: docs/guides/http-filters.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ The request object has the following shape:
 ```typescript
 type Request = {
   method: string
-  url: string
+  uri: string
```
Comment (tusharmath) [inline docs/guides/http-filters.md]: uri is not a string any more. Please check again.

**Comment by guptadeepak8** [comment]
@tusharmath should i add more additional fields from the Uri struct  Request

File: docs/guides/http-filters.md (hunk 1 above)
Comment (guptadeepak8) [reply docs/guides/http-filters.md]: done

File: docs/guides/http-filters.md [hunk 2]
```diff
@@ -102,7 +102,13 @@ The request object has the following shape:
 ```typescript
 type Request = {
   method: string
-  url: string
+  uri: {
+    path: string
+    query: {[key: string]: string}
```
Comment (tusharmath) [inline docs/guides/http-filters.md]:
```suggestion
    query?: {[key: string]: string}
```

**Review by tusharmath — APPROVED** [review]

**Review by tusharmath — APPROVED** [review]

**Review by tusharmath — APPROVED** [review]

**Pull request merged**